support/support.go

```diff
@@ -61,84 +62,141 @@ type Deps struct {
 }
 
 func DeploymentInfo(ctx context.Context, client *codersdk.Client, log slog.Logger) Deployment {
-	var d Deployment
-
-	bi, err := client.BuildInfo(ctx)
-	if err != nil {
-		log.Error(ctx, "fetch build info", slog.Error(err))
-	} else {
+	var (
+		d  Deployment
+		m  sync.Mutex
+		wg sync.WaitGroup
+	)
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		bi, err := client.BuildInfo(ctx)
+		if err != nil {
+			log.Error(ctx, "fetch build info", slog.Error(err))
+			return
+		}
+		m.Lock()
```

Comment:
We don't need a mutex here as each routine is assigning to their own field.

Reply:
Removed